core/chains/legacyevm/chain.go

```diff
@@ -393,6 +394,16 @@ func (c *chain) SendTx(ctx context.Context, from, to string, amount *big.Int, ba
 	return c.Transact(ctx, from, to, amount, balanceCheck)
 }
 
+func (c *chain) LatestHead(_ context.Context) (types.Head, error) {
+	latestChain := c.headTracker.LatestChain()
```

Comment:
`latestChain` [could be nil](https://github.com/smartcontractkit/chainlink/blob/f202bcfe45648cc803b38650a7aaf6fecb91969d/core/chains/evm/headtracker/head_saver.go#L68) and this would cause a panic when trying to access properties

```suggestion
	latestChain := c.headTracker.LatestChain()
	
	if latestChain == nil {
		return types.Head{}, nil
	}
```

Reply:
resolved